ocs_ci/helpers/dr_helpers.py

```
@@ -3038,9 +3038,26 @@ def create_ingress_cert_dr(
+            existing_certs = (
+                user_ca_bundle_cm.get().get("data").get("ca-bundle.crt", "").strip()
+            )
+        else:
+            existing_certs = ""
+        if existing_certs:
+            ssl_data.append(existing_certs + "\n")
+
+        default_ingress_cert_data = default_ingress_cert.get()["data"][
+            "ca-bundle.crt"
+        ].strip()
+        if not (default_ingress_cert_data in existing_certs):
```

Ruff (0.15.20)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)